compiler/src/dotty/tools/dotc/semanticdb/ExtractSemanticDB.scala

```diff
@@ -283,6 +283,14 @@ class ExtractSemanticDB extends Phase:
 
     end PatternValDef
 
+    extension (tree: NamedDefTree):
+      private def adjustedNameSpan(using Context): Span =
```

Comment:
Couldn't this be done in `nameSpan` itself?

Reply:
I was not sure if it was always meant to represent what is visible in the source code, but seeing as it affects printers and is empty for synthetics I would say it makes sense

Reply:
I made the change but it does also require adding `Context` parameter to `nameSpan`